notebooks/features/simple_deep_learning/DeepLearning - Deep Vision Classifier.ipynb

```diff
@@ -0,0 +1,186 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Deep Learning - Deep Vision Classifier"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Environment Setup -- reinstall horovod based on new version of pytorch"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install https://mmlspark.blob.core.windows.net/pip/CUR_VERSION/synapseml_deep_learning-0.10.0.dev1-py2.py3-none-any.whl --force-reinstall --no-deps"
```

Comment:
Is this necessary with the latest stuff to make it packaged along with python jar?

Reply:
Yes, see my comment on GPUtests :D